### DISCUSSION TITLE
Resizing graph pane

### DISCUSSION THREAD

**Discussion by IainA42** [blackadderkate/weather-widget-2#132] [Q&A] [answered]
When I open the graph pane, the axes along the bottom indicate 4 days of data while the graph shows only one day.  In the previous version I could resize the pane - is it possible in this version?

**Comment by Cris70** [comment]
I want to resize the pane too. Since a few weeks, the pane has become smaller, so the controls are all cramped and overwriting each other.

**Reply by tonhogen** [reply]
This issue where the graph pane is basically cut, appeared for the first time when I updated KDE to version 5.26: something in KDE has changed that causes this issue.

**Comment by Cris70** [comment]
This is now fixed for me

**Reply by IainA42** [reply]
So how was this sorted for you?  What did you change?

**Comment by Cris70** [comment, marked as answer]
I was not on the latest release. It was 2.3.1 IIRC. After updating to the latest (2.3.4) and a reboot, it was fixed.

**Reply by tonhogen** [reply]
Same here - updated to 2.3.3  and it's all good now!
